votes/initiateNewVote/_EDIT_ME.yml

```
@@ -24,8 +24,12 @@ headerInstructions: |
 #    voters express their preference for each candidates, no matter how many
 #    there are.
 candidates:
-  - TODO
-  - TODO
+  - Throw when passed to `--env-file` flag, provide no other flag (status quo).
```

Can we add another option

```suggestion
  - Do nothing (this is the same behavior as `dotenv` and every other library and programming language supporting this).
```

(Assuming there isn't another library or language with different behavior I'm just not aware of)

---

(I'm also happy with different phrasing)

---

I don't think this is correct. `dotenv` does not throw error if file is not found, but we throw an error right now.

---

Can you rephrase "do nothing" to "don't warn and don't throw an error"?

---

```suggestion
  - Throw when passed to `--env-file` flag, provide no other flag (status quo).
  - Ignore when passed to `--env-file` flag, provide no other flag.
```

---

I think we should make it clear that: `(this is the same behavior as `dotenv` and other libraries and platforms supporting .env)`

---

I am fine with any phrasing that makes it explicit that this option is the one that isn't "we picked an API that reached maturity through userland and we're sticking with the design our community adopted rather than deciding to change its design by committee".

I honestly don't have a strong opinion about whether we should throw or warn or ignore by default but I trust our community and ecosystem to adopt the option with better developer experience. The fact the ~10 implementations I checked* reached the same conclusion and it's different from all our options is troubling to me.


* (and I may be wrong and there are others who reached a different conclusion)

---

This is and has been the behavior for a while though you can add a config option (verbose:true or debug:true) in order to change it to warn. At least that is my understanding